Lib/asyncio/timeouts.py

```diff
@@ -34,9 +35,21 @@ def __init__(self, when: Optional[float]) -> None:
         self._when = when
 
     def when(self) -> Optional[float]:
+        """Return the current deadline, or `None` if the current deadline is not set.
+
+        The deadline is a float, consistent with the time returned by `loop.time()`.
```

Comment:
The type annotation seems to already say this, I don't think mentioning float is necessary.

Reply:
Agreed -- just the first sentence is fine.